README.md

```diff
@@ -73,6 +73,7 @@
 - [Sass Flexible Grid System](http://flexible.gs/install/sass) - Sass flexible grid system.
 - [Susy](https://github.com/oddbird/susy) - Responsive layout toolkit for Sass.
 - [Avalanche](http://colourgarden.net/avalanche) - Lightweight, responsive, Sass-based, BEM-syntax grid system.
+- [Toast Grid](http://daneden.github.io/Toast/) - A flexible and lightweight grid framework by the creator of animate.css
```

Comment:
- Can you please change the link title to be `[Toast](...) - Description here.` to follow its original name?
- Can you please end the description sentence with a full stop as stated in [contributing.md](https://github.com/Famolus/awesome-scss/blob/master/contributing.md)?